config/base/derive/src/utils.rs

```diff
@@ -197,6 +197,7 @@ pub struct StructWithFields {
     pub env_prefix: String,
     pub vis: syn::Visibility,
     _struct_token: Token![struct],
+    pub parent_ident: Option<Ident>,
```

Comment:
I think we should keep this struct single purpose (smt we receive as macro input).

Reply:
Fair enough, maybe I overoptimised thoughtlessly.